packages/app-store/applecalendar/api/add.ts

```diff
@@ -18,9 +18,29 @@ export default async function handler(req: NextApiRequest, res: NextApiResponse)
       select: {
         email: true,
         id: true,
+        credentials: {
+          where: {
+            type: "apple_calendar",
+          },
+        },
       },
     });
 
+    let alreadyExistsId = -1;
+
+    for (const credential of user.credentials) {
```

Comment:
@Amit91848 had a question for you, is there any added performance advantage or benefit for using the `for of` loop here instead of your previous approach where you were using `array.find()` before? I found the previous approach which you took much more clean and readable, and I just feel like somewhere in the future if we would want to modify this code, it could get messy real fast and that it might lead to a refactor then. Other than that I have no doubts over the code, LGTM and both the approaches should work as expected so kudos for that!

Reply:
It was just easier to break out of loop when using for. Changed it back to find 👍